metricbeat/module/windows/perfmon/defs_pdh_windows.go

```diff
@@ -200,9 +200,9 @@ const (
 	// PdhFmtDouble returns data as a double-precision floating point real.
 	PdhFmtDouble PdhCounterFormat = C.PDH_FMT_DOUBLE
 	// PdhFmtLarge returns data as a 64-bit integer.
-	PdhFmtLarge = C.PDH_FMT_LARGE
+	PdhFmtLarge PdhCounterFormat = C.PDH_FMT_LARGE
 	// PdhFmtLong returns data as a long integer.
-	PdhFmtLong = C.PDH_FMT_LONG
+	PdhFmtLong PdhCounterFormat = C.PDH_FMT_LONG
```

Comment:
Looks like other formatters below also should have the same `PdhCounterFormat` type applied.